subsys/net/lib/download_client/src/download_client.c

```diff
@@ -680,13 +708,11 @@ int download_client_init(struct download_client *const client,
 int download_client_connect(struct download_client *client, const char *host,
```

Comment:
Does it make still make sense to have separate `_connect()` and `_start()` functions? Or maybe we can rename this function?

Reply:
I did not plan to do any API changes.